### PR TITLE
feat: lemma resources route

### LINKED ISSUE #166: API routes to load lexicon data
* Add a new route `GET /lemma/:lemmaId/resources`
* This route is going to return all lexicon entries for a certain lemma
* Return data in the following shape:
```
{
  data: {
    resource: string
    entry: string
  }[]
}
```

### PR DESCRIPTION
**Pull request by Pertempto**
## What has changed

Added the lemma resources API route and documentation.

## Connected Issues

Closes #166 

## QA Steps

- [ ] Add a row or a few rows to the `LemmaResource` table
- [ ] Query for that data using the route `/api/lemmas/[lemmaId]/resources`

## Post-Deployment

- [ ] Nothing required

### CODE REVIEW

File: packages/api/public/spec/schemas/parameters.yml [hunk 1]
```diff
@@ -30,3 +30,11 @@ userId:
   schema:
     type: string
   example: '123e4567-e89b-12d3-a456-426614174000'
+lemmaId:
+  name: lemmaId
+  in: path
+  description: TODO - add description here.
```
Comment (Pertempto) [inline packages/api/public/spec/schemas/parameters.yml]: I need a good description here. Any ideas?

**Comment by arrocke** [comment]
This is in draft mode. Are you wanting a full review, or just general feedback?

**Comment by Pertempto** [comment]
> This is in draft mode. Are you wanting a full review, or just general feedback?

I was asking for feedback. See the comment I left on the code

File: packages/api/public/spec/schemas/parameters.yml (hunk 1 above)
Comment (arrocke) [reply packages/api/public/spec/schemas/parameters.yml]: I'd just say something like `The id of the lemma as a strongs number`

**Comment by Pertempto** [comment]
It should be ready now.

**Review by arrocke — APPROVED** [review]
Looks good

**Pull request merged**